az-digital/az_quickstart
# Close #3186 Use Composer to track az-icons package,  and  add ability for site owner to choose how az-icon library is loaded.

## Description
In this PR I am 
1. Adding a new github action that utilizes repository dispatch paradigm to create a new PR upon new release of az-icons. (this matches what we do for [arizona-bootstrap](https://github.com/az-digital/az_quickstart/blob/main/.github/workflows/arizona-bootstrap-release.yml))
2.Removing the copy of the icon font from this repository
3. Adding theme settings and defaults for controlling the output of the library.
4. Update composer.json so that it downloads az-icons


## Related issues
Close #3186

## How to test
Navigate to theme settings for az_barrio
Adjust the settings for az-icons 
<img width="1635" alt="image" src="https://github.com/az-digital/az_quickstart/assets/1023167/7a4c9927-7427-434b-89d6-5df3eae50084">
Save
Navigate to front end, and check that your settings are being respected.

## Types of changes

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
